Makefile

```diff
@@ -162,6 +162,16 @@ format-md: ## Format all markdown and MDX files to wrap at 80 characters
 lint: ## Run linter (clippy)
 	cargo clippy --all-targets -- -D warnings --allow clippy::mutable_key_type
 
+.PHONY: lint-bash
+lint-bash: ## Check all shell scripts using shellcheck
```

Comment:
This works for me. 

Should we also add `shellcheck` into `install-system-deps.sh`? I'd definitely add it to the `flake.nix` since it's something that developers might want to run locally.